h2/test_h2_headers.py

```diff
@@ -106,3 +107,91 @@ def test_random_header(self):
         self.start_tempesta()
         self.start_all_clients()
         self.wait_while_busy(curl)
+
+class CurlTest(tester.TempestaTest):
+
+    clients = [
+        {
+            'id' : 'curl',
+            'type' : 'external',
+            'binary' : 'curl',
+            'cmd_args' : (
+                '-k '
```

Comment:
`-f` key might be handy here, Normally curl return `0` if a response is received, with the key it returns non-null code on 4xx and 5xx responses, and the test will be marked as failed.